src/index.html

```diff
@@ -3,9 +3,38 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>Leaderboard </title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <h1>Leaderboard</h1>
+    <div class="main">
+        <div class="scores">
+            <div class="header">
+                <h2 class="recent">Recent Scores</h2>
+                <button type="button" class="refresh">Refresh</button>
+            </div>
+            <ul class="list">
+                <li>Name: 100</li>
+                <li>Name: 90</li>
+                <li>Name: 80</li>
+                <li>Name: 70</li>
+                <li>Name: 60</li>
+                <li>Name: 50</li>
+                <li>Name: 40</li>
+                <li>Name: 30</li>
+            </ul>
+        </div>
```

Comment:
- [ ] Kindly remove this HTML placeholder since we're sending and receiving data from/to the Leaderboard API. 🚮